validate.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancher/storage/docker/volumeplugin"
+	"net/http"
+)
+
+var (
+	validatePort = ":1000"
+	checkOnInit  = true
+	driver       *volumeplugin.RancherStorageDriver
+)
+
+func startValidation(d *volumeplugin.RancherStorageDriver) {
```

Comment:
This comment is for NFS part of validation. It's kind of annoying if you're going to use mount to check the connectivity of NFS everytime, even it's already mounted to another directory. Is there a better way?